pydantic/typing.py

```diff
@@ -58,7 +58,7 @@
     def evaluate_forwardref(type_: ForwardRef, globalns: Any, localns: Any) -> Any:
         return type_._evaluate(globalns, localns)
 
-elif sys.version_info < (3, 12):
+elif sys.version_info < (3, 12, 8):
```

Comment:
```suggestion
elif sys.version_info < (3, 12, 4):
```

Reply:
type_params was only added for 3.12.8, no?

Reply:
Ah... guess I misread the bug description. Have amended and force-pushed your suggestion.